cmd.go

```diff
@@ -264,13 +272,20 @@ type Uploader interface {
 	Upload(ctx context.Context, image io.ReadSeeker, size int64) (refs []string, retErr error)
 }
 
-func parseConfigFiles() (*config.ConfigFile, error) {
+func parseConfigFiles(configPath string) (*config.ConfigFile, error) {
+	configLocation := configName
+	configDirLocation := configDir
+	if configPath != "" {
+		configLocation = path.Join(configPath, configName)
+		configDirLocation = path.Join(configPath, configDir)
+	}
```

Comment:
```suggestion
	configLocation := path.Join(configPath, configName)
	configDirLocation := path.Join(configPath, configDir)
```
Will just return the default path if configPath is empty